models/tags.js

```diff
@@ -0,0 +1,57 @@
+const db = require('../db');
+
+class tags {
```

Comment:
Convention is to name your classes or models with capital first letters. When you have `Tag` or `Tags`, consider what it will look like when you use your static methods. In the other file we can see `tags.createTag`, this is where having `tag` in the name is redundant. `Tags.create` is a bit more concise!